packages/connect/src/types/settings.ts

```diff
@@ -37,4 +37,5 @@ export interface ConnectSettings {
     proxy?: Proxy;
     sharedLogger?: boolean;
     useCoreInPopup?: boolean;
+    keepalive?: boolean;
```

Comment:
@mroz22 What do you think? maybe we should name it `keepaliveWebextension`? but it looks too long. Not sure.

Reply:
yes, we must speak now or keep silent forever, this is part of public API 🤔.

`keepalive` vs `keepAlive` vs `keepaliveWebextension`, `keepAliveServiceWorker` hmmmm. what does @Hannsek think?

Reply:
Also I like putting comments like this

```
/* keepalive features does this and that */
keepalive?: boolean;
```

This way when I hover over this field anywhere across the codebase, vscode gives me a hint what it is.